elastic/kibana
# [Response Ops][Alerting] Allow runtime fields to be selected for Elasticsearch query rule type group by or aggregate over options

Resolves https://github.com/elastic/kibana/issues/157258

## Summary

Gets the runtime_mappings from the es query, and includes them in the query to retrieve the fields.
Also gets runtime mappings from data views.


### Checklist



### To verify

- Create a new Es Query rule using DSL
- Include runtime mappings in your query and verify that the runtime fields are listed in the group by or aggregate over options below the query
- Create a dataview and include runtime fields
- Create a new Es Query rule using KQL
- Verify that the runtime fields are listed in the group by or aggregate over options below the query
